composer/checkpoint/load.py

```diff
@@ -139,7 +139,10 @@ def load_checkpoint(
         assert model is not None
         assert model_child_path is not None
         model_load_path = os.path.join(load_path, model_child_path)
-        load_model_checkpoint(model, load_path=model_load_path, load_options=load_options)
+        if state:
```

Comment:
```suggestion
        if state is not None:
```
I don't think this is correct -- you should set return values if state is passed in, but you should still call `load_model_checkpoint` even if state is not specified

Reply:
let me know if my fix changes things